modules/storage-persistent-storage-pv.adoc

```diff
@@ -139,29 +139,27 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 endif::[]
 |===
 --
-1. ReadWriteOncePod access mode for persistent volumes is a Technology Preview feature.
+1. RWOP uses the SELinux mount feature. Supported vendors include EBS, Azure Disk, GCP PD, IBM VPC Block, Cinder, and vSphere.
```

Comment:
RWOP uses the SELinux context mount feature.
Maybe add ODF as well.

Reply:
Suggest to update as  "include AWS EBS xxx".

Reply:
confirmed with the ODF PM, they want us to add openshift data foundation to the list

Reply:
Done.